.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "recommendations": [
+    // meta-package for vscode which includes
+    // * ms-vscode.cpptools - provides c++ language support for vscode
+    // * ms-vscode.cmake-tools - provides CMake build support for vscode
+    "ms-vscode.cpptools-extension-pack",
+
+    // uses .editorconfig file to apply e.g. 
+    // tab indentation on sc files and space indentation elsewhere
+    "EditorConfig.EditorConfig",
+
+    // syntax highlighting in sc/schelp files for vscode
+    "jatinchowdhury18.vscode-supercollider",
+  ],
+}
```

Comment:
Purely cosmetic: this file now uses 2-space indentation, while other json files use 4 spaces. Could go in either as is or be changed for consistency.

Reply:
thx for the hint - i did not format those properly as our `.editorconfig` suggests using 2 spaces for json indentation

https://github.com/supercollider/supercollider/blob/916b10bb59a5020155cc7fa45758bbd9b307645e/.editorconfig#L29-L31

I now applied the 2 space formatting to the other .json files of this PR.